run/test/specs/onboarding_pp.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { bothPlatformsIt } from '../../types/sessionIt';
+import { PrivacyPolicyButton, SplashScreenLinks } from './locators/onboarding';
+import { runOnlyOnAndroid, runOnlyOnIOS } from './utils';
+import { closeApp, openAppOnPlatformSingleDevice, SupportedPlatformsType } from './utils/open_app';
+
+// risk assessment for later: high, because of the legal nature of these terms 
+bothPlatformsIt('Onboarding privacy policy', onboardingPP)
+
+const ppURL = 'https://getsession.org/privacy-policy';
```

Comment:
For consistency throughout the tests, all variables should be declare inside the test

Reply:
done